pkgs/by-name/tr/traderepublic-portfolio-downloader/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+buildGoModule rec {
+  pname = "traderepublic-portfolio-downloader";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "dhojayev";
+    repo = "traderepublic-portfolio-downloader";
+    tag = "v${version}";
+    sha256 = "sha256-tlrIYAJu8RXcvESKbdtJw4bkB6I+/gtvwAfjho8Fpt0=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/xadhamzz7bx2w46h34gs36vs1cm4965v-source.drv':
         specified: sha256-tlrIYAJu8RXcvESKbdtJw4bkB6I+/gtvwAfjho8Fpt0=
            got:    sha256-EXMG+53KKCTbuTLiybO9Zwsog8oOkd1Jrp8+eIFDYoE=

```

Reply:
That's true, just fixed it. Surprised me that it used the old fixed output derivation though the tag failed. Guess I should have a look at how fetchFromGitHub is implemented.